apps/condo/domains/ticket/hooks/useTicketCancelModal.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { CSSProperties, useCallback, useMemo, useState } from 'react'
+import { Modal, Typography } from 'antd'
+import { Button } from '@app/condo/domains/common/components/Button'
+import { useIntl } from '@core/next/intl'
+
+type useTicketCancelModalType = (updateTicket: (id: string) => void) => { openModal: (statusCanceledId: string) => void, cancelTicketModal: JSX.Element, closeModal: () => void }
+
+const MODAL_BUTTON_CANCEL_STYLE: CSSProperties = { fontSize: 16 }
+
+export const useTicketCancelModal: useTicketCancelModalType = (updateTicket) => {
+    const intl = useIntl()
+    const cancelModalTitle = intl.formatMessage({ id: 'pages.condo.ticket.id.CancelModal.title' })
+    const cancelButtonLabel = intl.formatMessage({ id: 'pages.condo.ticket.id.CancelModal.cancelButtonLabel' })
+    const cancelModalContent = intl.formatMessage({ id: 'pages.condo.ticket.id.CancelModal.content' })
```

Comment:
To conform code style in other places:

```suggestion
    const CancelModalTitleMessage = intl.formatMessage({ id: 'pages.condo.ticket.id.CancelModal.title' })
    const CancelButtonLabelMessage = intl.formatMessage({ id: 'pages.condo.ticket.id.CancelModal.cancelButtonLabel' })
    const CancelModalContentMessage = intl.formatMessage({ id: 'pages.condo.ticket.id.CancelModal.content' })
```